plugin/pkg/chain/chain.go

```diff
@@ -1,15 +1,19 @@
 package chain
 
-// PostRequest is the common struct containing the body and url
-type PostRequest struct {
+import "github.com/ugorji/go/codec"
+
+// HttpData is the common struct containing the body and url
+// Tag is for specifying the response message
+type HttpData struct {
```

Comment:
Maybe we need Method (GET/POST/DELETE/OPTION/PUT) in HttpData. For my understanding, COSMOS use `GET` method when the broadcast_tx: https://docs.tendermint.com/v0.35/rpc/#/Tx

Reply:
Agree. I added method field.